sycl/source/CMakeLists.txt

```diff
@@ -34,6 +46,21 @@ function(add_sycl_rt_library LIB_NAME)
       PRIVATE ${OpenCL_LIBRARIES}
       PRIVATE ${CMAKE_DL_LIBS}
   )
+  # Splitting xpti out so the dependencies can be defined
+  # for windows and the rest
+  if (MSVC)
+    # MSVC in particular will use the additional
+    # optional argument to link in the debug version
+    # correctly
+    target_link_libraries(${LIB_NAME}
+        PRIVATE ${ARG_XPTI_LIB}
+    )
+  else()
+    target_link_libraries(${LIB_NAME}
+        PRIVATE ${ARG_XPTI_LIB}
+        PRIVATE dl
+    )
+  endif()
```

Comment:
Please, merge this if with one which is on line 28

Reply:
Done!
@romanovvlad  Please take a look at the CMakeLists.txt - it should have the necessary changes along with diagnostics when LLVM_EXTERNAL_XPTI_SOURCE_DIR is not defined.